### PR TITLE
Fix: Newsletter Component Updated

### PR DESCRIPTION
**Pull request by SatyamVyas04**
This pull request fixes issue #234

### CODE REVIEW

**Comment by SatyamVyas04** [comment]
@hari 🙌🏻

**Comment by hari** [comment]
@SatyamVyas04 Did you test it? The `success` and `error `state will probably not work because you are using two different instances of the hook.

**Comment by SatyamVyas04** [comment]
> @SatyamVyas04 Did you test it? The `success` and `error `state will probably not work because you are using two different instances of the hook.

I tried adding myself to the newsletter earlier and indeed, couldn't see the two working, I'll look into this and add another commit, fixing the same. Would love any edits and changes from yourside.

**Comment by SatyamVyas04** [comment]
image

This seems to work 🚀

**Comment by SatyamVyas04** [comment]
@hari let me know if there are any other changes.

File: app/_landing/newsletter.tsx [hunk 1]
```diff
@@ -1,55 +1,78 @@
...
-          ""
-        ) : (
-          "100% free. No spam. No noise. Unsubscribe at any time."
-        )}
-      </p>
-    </div>
+        {isLoading && <Loader2 className="mr-2 h-4 w-4 animate-spin" />}
+        {isLoading ? "Please wait" : "Subscribe"}
+      </Button>
+    </form>
```
[the remark this answers is not in the document]
Comment (SatyamVyas04) [reply app/_landing/newsletter.tsx:42]: Sure, will get it done 👍🏻

**Comment by SatyamVyas04** [comment]
@hari worked on the requested changes, simplifying the form submission logic while keeping the UI unchanged.

**Review by hari — APPROVED** [review]

**Pull request merged**